backend/blueprints/opml.py

```
@@ -178,12 +178,12 @@ def _get_autosave_directory():
                 abs_db_path = os.path.abspath(db_path)
                 data_dir = os.path.dirname(abs_db_path)
                 logger.debug(
-                    "Resolved autosave directory from SQLite path: %s", data_dir
-                )
+                    "Resolved autosave directory from SQLite path: %s",
+                    data_dir)
             except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)